3_calibrate/src/calibration_utils.R

```diff
@@ -93,10 +117,9 @@ run_glm_cal <- function(nml_obj,
 #'
 
 set_eval_glm <- function(par, caldata_fl, sim_dir, nml_obj){
```

Comment:
Since you are using `set_eval_glm` to evaluate the model (both inside a calibration routine but also with default params), you might want to consider naming the input `evaldata_fl` instead of `caldata_fl` to avoid confusion.

Reply:
done!